tests/Integration/Endpoints/RecommendationsProxyEndpointTest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Parsely Recommendations API Proxy Endpoint tests.
+ *
+ * @package Parsely\Tests
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\Integration;
+
+use Parsely\Parsely;
+use Parsely\Endpoints\Recommendations_API_Proxy;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Server;
+use WP_Test_REST_Controller_Testcase;
+
+/**
+ * Parsely REST API tests.
+ */
+final class RecommendationsProxyEndpointTest extends WP_Test_REST_Controller_Testcase {
```

Comment:
I'm attempting to extend our `TestCase` so we don't have so many skipped tests d/t having to implement all the abstract methods of `WP_Test_REST_Controller_Testcase`, but it's not immediately obvious why it's not working.